code/obj/machinery/computer/cloning.dm

```diff
@@ -72,7 +72,7 @@
 
 /obj/machinery/computer/cloning/New()
 	..()
-	SPAWN_DBG(0.5 SECONDS)
+	SPAWN_DBG(0.7 SECONDS)
```

Comment:
Out of the scope of this PR but this stuff should move to `initialize` as that's guaranteed to run _after_ the map loads and SPAWN_DBG hacks can easily fail if the server is having a bad day.

Reply:
Aahh... I did this because I kept getting runtimes where `scanner` kept returning null while the map was initializing. I do not know how to move this to `initialize` as I didn't even know that was a thing, lol.

Reply:
It's fine, I'll do it later. (But it's simple!)